src/pages/home/report/ReportActionsView.js

```diff
@@ -80,14 +82,105 @@ const defaultProps = {
     },
 };
 
-function ReportActionsView(props) {
+const DIFF_BETWEEN_SCREEN_HEIGHT_AND_LIST = 120;
```

Comment:
This is mostly for web and desktop since they usually have big screens that can handle larger list sizes. In these areas, we generally use pixels for measurements. Using 'dp' could work too, but pixels are the standard for web and desktop development, so sticking with pixels should be fine and not cause any problems